integration/docker/csi/alluxio/nodeserver.go

```diff
@@ -246,8 +240,15 @@ func getAndCompleteFusePodObj(nodeId string, req *csi.NodeStageVolumeRequest) (*
 	// Set fuse mount options
 	fuseOptsStr := strings.Join(req.GetVolumeCapability().GetMount().GetMountFlags(), ",")
 	csiFusePodObj.Spec.Containers[0].Args = append(csiFusePodObj.Spec.Containers[0].Args, "--fuse-opts=" + fuseOptsStr)
-        csiFusePodObj.Spec.Containers[0].Args = append(csiFusePodObj.Spec.Containers[0].Args, req.GetStagingTargetPath())
-        csiFusePodObj.Spec.Containers[0].Args = append(csiFusePodObj.Spec.Containers[0].Args, targetPath)
+
+	// Set fuse mount point
+	csiFusePodObj.Spec.Containers[0].Args = append(csiFusePodObj.Spec.Containers[0].Args, req.GetStagingTargetPath())
+
+	// Set alluxio path to be mounted if set
+	alluxioPath := req.GetVolumeContext()["alluxioPath"]
```

Comment:
With the new changes to `entrypoint.sh` and `alluxio-fuse`, if `alluxioPath` is not set in the Fuse k8s yaml file,  `alluxio.fuse.mount.alluxio.path` property shall be used for the `alluxioPath`. So we are not going to set it to `/` if `alluxioPath` is not set here.

@Haoning-Sun FYI